.github/workflows/coveralls.yml

```diff
@@ -1,26 +1,28 @@
-on: ["push", "pull_request"]
+on: 
```

Comment:
this is ought to remove repetition of running in case of updating the PR with new commits.